### PR TITLE
feat: run dashboard tile query

### PR DESCRIPTION
**Pull request by rephus**
Closes:  #4590

### Description:

Screenshot from 2023-03-22 10-40-17

### CODE REVIEW

File: packages/frontend/src/hooks/useQueryResults.tsx [hunk 1]
```diff
@@ -135,7 +154,7 @@ export const useSavedChartResults = (
     return useQuery<ApiQueryResults, ApiError>({
         queryKey,
         queryFn: () =>
-            getQueryResults({
+            getDashboardTileQueryResults({
```
Comment (owlas) [inline packages/frontend/src/hooks/useQueryResults.tsx:157]: Is the `useSavedChartResults` hook only used on dashboard tiles?

**Review by owlas — COMMENTED** [review]
see inline

File: packages/frontend/src/hooks/useQueryResults.tsx (hunk 1 above)
Comment (rephus) [reply packages/frontend/src/hooks/useQueryResults.tsx:157]: Yep 
image

**Review by owlas — APPROVED** [review]

**Pull request merged**